tests/05_message.ts

```diff
@@ -29,3 +39,50 @@ Deno.test({
     assertEquals(message.embeds[0].title, "Discordeno Test");
   },
 });
+
+Deno.test({
+  name: "[message] pin a message in a channel",
+  fn() {
+    pin(tempData.channelID, tempData.messageID);
+  },
+  ...defaultTestOptions,
+});
+
+Deno.test({
+  name: "[message] get pinned message in a channel",
+  async fn() {
+    const [msg] = await getPins(tempData.channelID);
+
+    // Assertions
+    assertExists(msg);
+    assertEquals(msg.id, tempData.messageID);
+    assertEquals(msg.pinned, true);
+  },
+  ...defaultTestOptions,
+});
+
+Deno.test({
+  name: "[message] unpin a message",
+  fn() {
+    unpin(tempData.channelID, tempData.messageID);
+  },
+  ...defaultTestOptions,
+});
+
+Deno.test({
+  name: "[message] add a reaction to a message",
+  fn() {
+    addReaction(tempData.channelID, tempData.messageID, "👍");
```

Comment:
Please test the <:name:id> version also somehow

Reply:
We should also test addReactions

Reply:
@Skillz4Killz, again, the `addReactions()` function `Promise.all()`s the `addReaction()` function.